pa11y-results.txt

```diff
@@ -0,0 +1,11 @@
+(node:65995) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
+(Use `node --trace-deprecation ...` to show where the warning was created)
+Running Pa11y on 6 URLs:
+ > http://localhost:6012/ - 0 errors
+ > http://localhost:6012/support - 0 errors
+ > http://localhost:6012/features - 0 errors
+ > http://localhost:6012/pricing - 0 errors
+ > http://localhost:6012/documentation - 0 errors
+ > http://localhost:6012/accounts - 0 errors
+
+✔ 6/6 URLs passed
```

Comment:
Nice!  Does this file need to be added to `.git-ignore` or is it more like a `*lock` file of sorts and should be kept in source control for any future updates that might result in errors and therefore would be caught in review?